gammapy/cube/make.py

```diff
@@ -33,196 +33,313 @@ class MapMakerObs:
         Reference image geometry in true energy, used for IRF maps. It can have a coarser
         spatial bins than the counts geom.
         If none, the same as geom is assumed
-    exclusion_mask : `~gammapy.maps.Map`
-        Exclusion mask (used by some background estimators)
     migra_axis : `~gammapy.maps.MapAxis`
         Migration axis for edisp map
     rad_axis : `~gammapy.maps.MapAxis`
-        Radial axis for psf map
+        Radial axis for psf map.
+    cutout : bool
+        Whether to cutout the observation.
     """
 
     def __init__(
         self,
-        observation,
         geom,
         offset_max,
```

Comment:
If we move the max offset cut to a later step, it becomes useless to have it as an argument here. 
Then `cutout_size=5*u.deg` could be the default and setting it to `None` would result in no cutout performed.

Or is it for a later change?

Reply:
Maybe we wait for some more comments to #2451 and I make a follow up PR? But it would be definitely my preference to remove the mask safe computation...